### ISSUE TITLE
EZPlaceholderHook deprecated

### ISSUE THREAD

**Issue by SlimeDog** [A5H73Y/Parkour#145]
Spigot 1.13.2 final
PlaceholderAPI 2.10.0-b65
Parkour 5.2

Per [PlaceholderAPI](http://ci.extendedclip.com/job/PlaceholderAPI/changes):
```
#65 (May 5, 2019 10:40:16 AM)
Remove EZPlaceholderHook which has been deprecated for over a year. (commit: 6495a38) — Ryan McCarthy / githubweb
``` 
Placeholders seem to work, but...

On server start-up
```
[08:03:28] [Server thread/INFO]: [Parkour] [PlaceholderAPI] Successfully linked. Version: 2.10.0-DEV-65
[08:03:28] [Server thread/ERROR]: Error occurred while enabling Parkour v5.2 (Is it up to date?)
java.lang.NoClassDefFoundError: me/clip/placeholderapi/external/EZPlaceholderHook
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:1016) ~[?:?]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:132) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.JavaPluginLoader.getClassByName(JavaPluginLoader.java:200) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:93) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:82) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at me.A5H73Y.Parkour.Other.StartPlugin.setupPlaceholderAPI(StartPlugin.java:143) ~[?:?]
        at me.A5H73Y.Parkour.Other.StartPlugin.setupExternalPlugins(StartPlugin.java:43) ~[?:?]
        at me.A5H73Y.Parkour.Other.StartPlugin.run(StartPlugin.java:35) ~[?:?]
        at me.A5H73Y.Parkour.Parkour.onEnable(Parkour.java:40) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:347) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:410) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:436) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:350) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:580) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:542) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:420) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:294) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: me.clip.placeholderapi.external.EZPlaceholderHook
        at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:136) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:82) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        ... 24 more
```

**Comment by SlimeDog** [comment]
PlaceholderAPI dev [put it back temporarily](\https://github.com/PlaceholderAPI-Expansions/PlaceholderAPI/commit/75c3027e055867f9b1f1f529b743db679f5668e6), but you really need to update.

**Comment by A5H73Y** [comment]
Thanks for raising this, should be good to go now.
@steve4744 is going to check his plugins to see if any changes need to be made to support the update.

**Comment by SlimeDog** [comment]
should be good to go after next release. :)

**Comment by SlimeDog** [comment]
Confirmed fixed in Parkour 5.3-b287.
